src/process/ProcessManagerImpl.cpp

```diff
@@ -194,7 +194,12 @@ ProcessManagerImpl::shutdown()
         for (auto& pair : mProcesses)
         {
             // Mark it as "ready to be killed"
-            mKillable.push_back(pair.second);
+            auto killableIt =
```

Comment:
see my note on using a map instead: both `mKillable` and `mPending` want the same thing (and it's error prone)

Reply:
yeah, see above re: map